src/amo/components/AddonTitle/index.js

```diff
@@ -68,7 +71,11 @@ export const AddonTitleBase = ({
         <>
           {linkToAddon ? (
             <Link
-              to={addQueryParams(getAddonURL(addon.slug), { src: linkSource })}
+              to={addQueryParams(
+                getAddonURL(addon.slug),
+                { src: linkSource },
```

Comment:
This seems odd. It looks like the link will use `src` even if the `enableFeatureUseUtmParams` flag is on. I can see from the code of `addQueryParams` that this won't be the case, but the API/signature of this method is confusing as a result, imo.

Reply:
Yes, this is temporary, though. The clean-up patch won't be small but it will remove things like this by explicitly listing the UTM params. Because `src` is a special param used everywhere on the site and we want to replace it with UTM params, it was just easier to introduce the feature flag in this `addQueryParams` function, rather than trying to find all occurrences everywhere.

Reply:
So I took another look today and I am going to change that. We can pass `queryParamsForAttribution` instead of `linkSource`. That should be more explicit.